apps/backend/src/lambdas/parser.ts

```diff
@@ -65,15 +83,56 @@ export const handler = async (event?: unknown, _context?: Context) => {
 
       // TODO [CCFPCM-318] Excluded LABOUR2 files that are DDF, needs implementation
       const finalParseList = parseList.filter(
-        (filename) => !filename?.includes('LABOUR2')
+        (filename) => !filename?.includes('LABOUR')
```

Comment:
We may still need to account for these LABOUR2 files unless we are not getting these anymore?

Reply:
YES sorry, artifact of testing